src/main/java/net/glowstone/entity/GlowPlayer.java

```diff
@@ -2826,10 +2846,47 @@ public GlowBlock getDigging() {
         return digging;
     }
 
+    private void broadcastBlockBreakAnimation(GlowBlock block, int destroyStage) {
+        if (block == null) return;
```

Comment:
Agreed, changed to move the check outside the function - it should only be called with non-null block now